docs/docs/inline-environments.mdx

```diff
@@ -0,0 +1,132 @@
+---
+name: "Inline environments"
+route: "/inline-environments"
+menu: Advanced features
+---
+
+# Inline environments
+
+Inline environments is the practice of defining the environment's config inline
+for evaluation at runtime as opposed to configuring it statically in
+`spec.json`.
+
+## Converting to an inline environment
+
+Converting a traditional `spec.json` environment into an inline environment is quite
+straight forward. Based on the example from [Using Jsonnet](tutorial/jsonnet):
+
+The directory structure:
+
+```sh
+├── environments
+│   └── default # default environment
+│       ├── main.jsonnet # main file
+│       └── spec.json # environment's config
+├── jsonnetfile.json
+├── lib # libraries
+└── vendor # external libraries
+```
+
+The original files look like this:
+
+```jsonnet
+// main.jsonnet
+{
+  some_deployment: {/* ... */ },
+  some_service: {/* ... */ },
+}
+```
+
+
+```js
```

Comment:
```suggestion
```json
```

Reply:
I copy/paste that from your code. :grimacing: